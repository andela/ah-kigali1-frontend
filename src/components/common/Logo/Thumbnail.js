import React from "react";
import PropTypes from "prop-types";
import { Link } from "react-router-dom";
import img from "../../../assets/img/quill-drawing-a-line.svg";

const Thumbnail = ({ title = "AH" }) => (
  <Link className="col-md-6 col-sm-3 brand-name" to="/">
    <div className="brand">
      <img src={img} alt="logo" className="brand" />
    </div>
    <h3>{title}</h3>
  </Link>
);

Thumbnail.propTypes = { title: PropTypes.string };
Thumbnail.defaultProps = { title: "AH" };

export default Thumbnail;
