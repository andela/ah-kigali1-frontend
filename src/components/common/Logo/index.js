import React from "react";
import PropTypes from "prop-types";

const Logo = ({ className, ...props }) => (
  <img
    src={require("../../../assets/img/quill-drawing-a-line.svg")}
    alt="logo"
    className="logo"
    {...props}
  />
);

Logo.propTypes = {
  className: PropTypes.string
};

Logo.defaultProps = {
  className: "logo"
};
export default Logo;
