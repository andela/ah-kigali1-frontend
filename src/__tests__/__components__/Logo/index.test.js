import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Logo from "../../../components/common/Logo";

const warper = shallow(<Logo className="logo" />);
describe("Logo component", () => {
  it("matches the snapshot", () => {
    expect(toJson(warper)).toMatchSnapshot();
  });
});
