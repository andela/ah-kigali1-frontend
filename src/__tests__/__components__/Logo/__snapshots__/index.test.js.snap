// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component matches the snapshot 1`] = `
<img
  alt="logo"
  className="logo"
  src="test-file-stub"
/>
`;
